programs/sbf/Cargo.lock

```diff
@@ -6837,6 +6837,7 @@ dependencies = [
  "solana-transaction-metrics-tracker",
  "thiserror",
  "tokio",
+ "tokio-util 0.7.1",
```

Comment:
Because this is how dependency resolution works. Something in the dep graph requires <= 0.7.1 and because 0.7.12 and 0.7.1 are semver compatible it's using that. I didn't go and edit Cargo.lock - cargo did this change.

Reply:
Strange both are about solana-streamer's dependencies.